Objects/object.c

```diff
@@ -2161,6 +2162,9 @@ static PyTypeObject* static_types[] = {
     &PyODictKeys_Type,    // base=&PyDictKeys_Type
     &PyODictValues_Type,  // base=&PyDictValues_Type
     &PyODict_Type,        // base=&PyDict_Type
+
+    // Pyrona Region:
+    &Region_Type,
```

Comment:
Use same naming convention as the rest of the code?
```suggestion
    &PyRegion_Type,
```